issues-review

```diff
@@ -56,8 +56,8 @@ def close_issues(api, opts):
         count = len(issues)
         for issue in issues:
             labels = api.get("issues/%i/labels" % issue["number"])
-            enhancement = [l for l in labels if l["name"] == "enhancement"]
-            bug = [l for l in labels if l["name"] == "bug"]
+            enhancement = [la for la in labels if la["name"] == "enhancement"]
+            bug = [la for la in labels if la["name"] == "bug"]
```

Comment:
now it is "la la land" :smile:

Reply:
Is that because pycodestyle doesn't like the variable name `l`, as it's too ambiguous with `1`?

Reply:
https://www.flake8rules.com/rules/E741.html